data/json/monsters.json

```diff
@@ -2066,6 +2083,7 @@
     "armor_bash":12,
     "armor_cut":8,
     "vision_day":50,
+    "vision_night":5,
```

Comment:
OK, the brute and the hulk get 5 NV, superior to HNV.  I'm thinking that's a serious balance breaker since they're also the ones most lethal in melee,  whether or not being physically larger lets them see farther as a result of their eyes being higher up..